README.rst

```diff
@@ -26,9 +26,25 @@ Example
 
 The following query types are supported: A, AAAA, CNAME, MX, NAPTR, NS, PTR, SOA, SRV, TXT.
 
-The library supports both *asyncio* and *Trollius*.
+The library supports both *async/await*, *asyncio* and *Trollius*.
```

Comment:
Can you please undo this line? It mentions frameworks, and *async/await* is not one.